### ISSUE TITLE
mixup with larger alpha

### ISSUE THREAD

**Issue by hongyi-zhang** [dmlc/gluon-cv#427]
Hi, one of the authors of mixup here. Excited to see it being used in the latest pre-trained models!

Just curious, did you try using higher mixup for the largest models? For example, for resnet152_v1d it is likely that the model could benefit from stronger regularization (my guess is that alpha=0.4 may give some additional 0.3-0.4 acc boost).

**Comment by hetong007** [comment]
Welcome! 

I tried alpha=0.2 and alpha=1, the latter one worked on CIFAR but didn't really work on ImageNet. Will go ahead and test with alhpa=0.4

**Comment by hetong007** [comment]
So we have the results now, given `--mixup-alpha=0.4` we have the almost identical accuracy with `alpha=0.2`. It could be one of these reasons:

- We need larger `alpha`.
- We need a longer training to converge better.
- It is capped by the model capacity.

We'll work on pretrained models for ResNeXts/SENet so will probably test larger `alpha`.
